main.tf

```
@@ -28,6 +28,28 @@ resource "aws_vpc" "myvpc" {
   }
 }
 
+resource "aws_default_security_group" "default" {
+  vpc_id = aws_vpc.myvpc.id
+
+  ingress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
```

Please have it as a variable and limit to the IP address of your ISP
```suggestion
    cidr_blocks = [var.client_public_ip]
```

---

cidr does not take a single IP i had to update the whole network of my isp